src/routes/news/[slug].svelte

```diff
@@ -44,7 +54,31 @@
                 [BLOCKS.EMBEDDED_ASSET]: (node, next) => {
                     const asset = assetMap.get(node.data.target.sys.id);
                     return `<img src=${asset.url} alt="${asset.title}" />`;
-                }
+                },
+                [BLOCKS.EMBEDDED_ENTRY]: (node, next) => {
+                    // https://www.contentful.com/blog/2021/04/14/rendering-linked-assets-entries-in-contentful/
+                    // find the entry in the entryMap by ID
+                    const entry = entryMap.get(node.data.target.sys.id);
+
+                    // render the entries as needed by looking at the __typename 
+                    // referenced in the GraphQL query
+                    if (entry.__typename === "IFrame") {
+                        const iframeElementString = `
+                            <iframe
+                                src="${(entry as IFrameEntry).url}"
+                                height="${(entry as IFrameEntry).height}"
+                                width="${(entry as IFrameEntry).width}"
+                                frameBorder="0"
+                                scrolling="no"
+                                title="${(entry as IFrameEntry).title}"
```

Comment:
Try to follow the DRY principle and avoid casting the `entry` to `IFrameEntry` more than once.